### ISSUE TITLE
Some confusion about the experiment results

### ISSUE THREAD

**Issue by Udntknwme** [openvla/openvla#91]
Thanks for open-sourcing the great work! There are points in the paper that confuse me and I would like to ask for guidance as follows:

1. As stated in the paper, with all 7B parameters involved in training, why is the linguistic ability, in this case mainly the ability to follow the input instructions, not lost. (e.g. the experiment in the paper of placing jars on portrait pictures that did not appear in the training set) Is it because the number of rounds of training is not enough to lose the original linguistic ability, even for training with such a large amount of data as openx and with next token prediction accuracy converging to 95%?
2. In the recently added appendix (D.1), if training only a single dataset causes the success rate of the task to plummet by 30%, is this due to the fact that the lack of data in a single dataset results in a failure to fully converge on a single task so that other datasets of similar tasks are needed to supplement it?  Repeated training on the same dataset can be so much weaker than training across datasets is confusing.

Thanks again.

**Comment by kpertsch** [comment]
Hi thanks for the questions!
1. Note that we still have language inputs during finetuning on the robot data, so the mapping from language instruction to robot execution is exactly what we are finetuning for, thus the model does not "forget" it. We do see that the model forgets general conversational ability during finetuning (eg you can no longer ask it to caption an image), so your intuition here is correct. We have tried to co-train with some VLM datasets and this indeed enables the model to not forget conversational abilities while still being able to control robots well, so it's all a question of the data you use for pre-training.
2. When training on a single dataset only, it's not so much that the model can't fit the task, it's more that it very heavily overfits. We find that pre-training on a lot of robot data makes it much easier to then finetune the model on a small robot dataset. If you start directly with the small robot dataset the policies tend to work much worse.

Hope this helps answer some of your questions!
